### ISSUE TITLE
Display deployment health state

### ISSUE THREAD

**Issue by jandegen** [amosproj/amos2023ss04-kubernetes-inventory-taker#59]
## User story
1. As a user
2. I want / need a health state of a specific deployment
3. So that I can quickly see the deployment's current state on its detail page

## Acceptance criteria
- [ ] Display the state as both colour and text
- [ ] Text is always easily readable
- [ ] The health indicator is based on the PodStatus phase field
- [ ] Red on failed
- [ ] Green on succeeded
- [ ] Yellow on pending
- [ ] Blue on running
- [ ] Light gray on unknown or **paused**
- [ ] Dark gray on terminating

## Linked issues
- **Related to** #56 

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Colours match with colour scheme
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
